tgui/packages/tgui/interfaces/LC13AbnormalityArchive.js

```diff
@@ -0,0 +1,130 @@
+import { sortBy } from 'common/collections';
+import { flow } from 'common/fp';
+import { useBackend, useLocalState } from '../backend';
+import { Box, Button, LabeledList, Section, Stack } from '../components';
+import { Window } from '../layouts';
+import { capitalize } from 'common/string';
+import { createLogger } from '../logging';
+import { LabeledListItem } from '../components/LabeledList';
+
+const logger = new createLogger();
+
+export const LC13AbnormalityArchive = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    abnormality_info,
+  } = data;
+  const abnormality_by_name = flow([
+    sortBy(abnormality_info => abnormality_info.name),
+  ])(data.abnormality_info || []);
+  const [
+    currentAbnormality,
+    setcurrentAbnormality,
+  ] = useLocalState(context, 'currentAbnormality', null);
+  return (
+    <Window
+      width={800}
+      height={600}>
+      <Window.Content>
+        <Stack fill>
+          <Stack.Item width="150px">
+            <Section fill scrollable>
+              {abnormality_by_name.map(f => (
+                <Button
+                  key={f.name}
+                  fluid
+                  color="transparent"
+                  selected={f === currentAbnormality}
+                  onClick={() => { setcurrentAbnormality(f); }}>
+                  {f.name}
+                </Button>
+              ))}
+            </Section>
+          </Stack.Item>
+          <Stack.Item grow basis={0}>
+            <Section
+              fill
+              scrollable
+              title={currentAbnormality
+                ? capitalize(currentAbnormality.name)
+                : " Lobotomy Corporation Archive"}>
+              {currentAbnormality && (
+                <LabeledList>
+                  <LabeledList.Item label="File Info">
+                    {currentAbnormality.name}
+                  </LabeledList.Item>
+                  <LabeledList.Item>
+                    <LinesofText Subject={currentAbnormality} />
+                  </LabeledList.Item>
+                  <LabeledList.Item>
+                    <Button.Confirm
+                      fluid
+                      icon="check"
+                      color="red"
+                      textAlign="center"
+                      content="Print File"
+                      onClick={() => act('print_file', { ref: currentAbnormality.type })} />
+                  </LabeledList.Item>
+                </LabeledList>
+              )}
+            </Section>
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+// Theres a tower on this island.
+const LinesofText = (props, context) => {
+  const { Subject } = props;
+  const The_Tower = []; // The bottom of The_Tower
```

Comment:
Pascal_Snake_Case may look cool but please stick to CamelCase *or* snake_case

Reply:
Oh okay